src/libraries/Native/Unix/System.Globalization.Native/pal_timeZoneInfo.c

```diff
@@ -4,7 +4,10 @@
 //
 
 #include <stdint.h>
+
+#if defined(TARGET_UNIX)
 #include <unistd.h>
```

Comment:
Is this header actually needed on Unix?

Reply:
Not sure. I’ll double check and if not I’ll remove it.